tests/core/usage_statistics/test_usage_statistics.py

```diff
@@ -52,6 +64,141 @@ def in_memory_data_context_config_usage_stats_enabled():
+        "cli.init.create",
+        "cli.new_ds_choice",
+        "data_context.open_data_docs",
+        "data_context.build_data_docs",
+        "data_context.save.expectation.suite",
+        "data_context.add_datasource",
+    ]
+
+
+def test_init_message():
+    # record itself
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.__init__"][0],
+        usage_statistics_record_schema,
+    )
+    # non-empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.__init__"][0]["event_payload"],
+        init_payload_schema,
+    )
+
+
+def test_cli_suite_list_message():
+    # empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["cli.suite.list"][0],
+        usage_statistics_record_schema,
+    )
+
+
+def test_cli_suite_new_message():
+    # empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["cli.suite.new"][0],
+        usage_statistics_record_schema,
+    )
+
+
+def test_cli_new_ds_choice_message():
+    # record itself
+    jsonschema.validate(
+        valid_usage_statistics_messages["cli.new_ds_choice"][0],
+        usage_statistics_record_schema,
+    )
+    # non-empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["cli.new_ds_choice"][0]["event_payload"],
+        cli_new_ds_choice_payload,
+    )
+
+
+def test_cli_init_create_message():
+    # empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["cli.init.create"][0],
+        usage_statistics_record_schema,
+    )
+
+
+def test_data_context_open_data_docs_message():
+    # empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.open_data_docs"][0],
+        usage_statistics_record_schema,
+    )
+
+
+def test_data_context_build_data_docs_message():
+    # empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.build_data_docs"][0],
+        usage_statistics_record_schema,
+    )
+
+
+def test_data_context_save_expectation_suite_message():
+    # record itself
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.save.expectation.suite"][0],
+        usage_statistics_record_schema,
+    )
+    # non-empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.save.expectation.suite"][0][
+            "event_payload"
+        ],
+        save_or_edit_expectation_suite_payload_schema,
+    )
+
+
+def test_data_context_add_datasource_message():
+    # record itself
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.add_datasource"][0],
+        usage_statistics_record_schema,
+    )
+    # non-empty payload
+    jsonschema.validate(
+        valid_usage_statistics_messages["data_context.add_datasource"][0][
+            "event_payload"
+        ],
+        anonymized_datasource_schema,
+    )
+
+
+def test_add_datasource_usage_statistics(
```

Comment:
A test that validates the payload that comes from calling `add_datasource_usage_statistics()`